### ISSUE TITLE
Canva integration

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#847]
- People use canva to generate images for their events

**Comment by ivankocienski** [comment]
@katjam too vague

**Comment by katjam** [comment]
I rekon this os one of the old trello board issues I converted, but not specified by me. Agree though. If it's a thing, we can open another time.

**Comment by kimadactyl** [comment]
Just googled this as it came up in a chat with @honor-gfsc the other day and this looks like a trivial integration that could create a ton of value for people - reopening! Just having a "make a poster for this event in Canva" button would be super useful I think?

https://www.canva.com/button/

**Comment by honor-gfsc** [comment]
i agree this is important, i would like it if we can specify this further. this is something a partner mentioned to me as a desirable feature.

**Comment by katjam** [comment]
Removing won't fix label. @honor-gfsc if this is something your partners would like and use, please remove it from the dreamyard. When we scope this time, we'll create a place for feature requests validated by research for our various users.

**Comment by katjam** [comment]
Something similar came up in the Flourish research session where someone asked about generating the map/info poster Kim brought along that showcased a whole PlaceCal.

**Comment by honor-gfsc** [comment]
I don't know what Dreamyard means. This is something that came up as a feature people would like to see. I also remember talking about something along these lines with Kim and Rachael.

**Comment by katjam** [comment]
Dreamyard is this place https://github.com/geeksforsocialchange/PlaceCal/milestone/1 where we put stuff we think is a good idea but is not validated as a priority. It sounds like this is a legit feature request that we know will be meaningful for people now, so probably doesn't belong there.

**Comment by kimadactyl** [comment]
Potential route here: export a CSV that can be imported into a Canva template we make: https://www.canva.com/help/bulk-create/
